docs/content/install.md

```diff
@@ -11,9 +11,11 @@ We have a few release types available for you to use:
 
 You can also install and manage [mixins](#mixins) using porter,
 and use the [Porter VS Code Extension][vscode-ext] for help
-authoring bundles.
+authoring bundles. For Windows installation, you will need to
```

Comment:
Hmm, I see why you put this note here, because otherwise it would be copied 3 times on the page. However, people will link directly to the windows instructions like this:

https://deploy-preview-1088--porter.netlify.app/install/#latest-windows

and then they probably won't see the note. I recommend removing the message from this location and copy/pasting it into each windows installation section with the "For windows installation," removed since we are in the context of the windows section. 

I realize it seems silly to duplicate but hopefully it will make it more likely that they see the text.